docs/assets/js/swagger.yml

```diff
@@ -246,6 +246,10 @@ components:
           enum: [common_prefix, object]
         physical_address:
           type: string
+          description: |
+            The location of the object on the underlying object store.
+            Formatted as a native URI with the object store type as scheme ("s3://...", "gs://...", etc.)
+            Or, in the case of presign=true, will be an HTTP URL to be consumed via regular HTTP GET
```

Comment:
Is it related to deprecation?

Reply:
No, just a missing "make gen".  I'll rebase before pushing.  The automated generated copy of Swagger.yml must be generated automatically.